Manifests/index

```diff
@@ -1751,6 +1751,17 @@
 			<key>version</key>
 			<integer>3</integer>
 		</dict>
+		<key>com.apple.sso</key>
+		<dict>
+			<key>hash</key>
+			<string>485e486df076b7a9c7c6a774d93086af</string>
+			<key>modified</key>
+			<date>2019-10-18T16:49:24Z</date>
+			<key>path</key>
+			<string>Manifests/ManifestsApple/com.apple.sso.plist</string>
+			<key>version</key>
+			<integer>1</integer>
+		</dict>
```

Comment:
In the future, would you mind not updating the manifest indexes as part of your commits?  That way if we're merging multiple or different PRs first we avoid a commit conflict.  Whomever is doing the PR merges should be the one to update the indexes.

Reply:
You got it @apizz.